profiles/car.lua

```diff
@@ -171,13 +171,41 @@ function setup()
       'minor',
     },
 
-    route_speeds = {
-      ferry = 5,
-      shuttle_train = 10
-    },
-
-    bridge_speeds = {
-      movable = 5
+    routes = {
```

Comment:
If I'm reading this correctly, we now would allow routing on trains in the car profile? I think using public transportation is rather unexpected and leads to weird/impossible routes (car -> train -> car).

What about keeping the default here of just using ferries/movable bridges?

Reply:
yeah i agree, trains shot not be routable by deafult. only thing we might want by default is shuttle train? https://github.com/Project-OSRM/osrm-backend/blob/4297d7f56babc3302d0bc6a42cc0011a255d0a40/profiles/car.lua#L195

Reply:
Yes `shuttle_train` makes sense to me.